app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -42,6 +51,11 @@ trait UserRoutes extends Authentication
       pathEndOrSingleSlash {
         get { getUserByEncodedAuthId(authIdEncoded) } ~
         put { updateUserByEncodedAuthId(authIdEncoded) }
+      } ~
+      pathPrefix("dropbox-setup") {
```

Comment:
It probably makes sense to not have the user id in the route, since you can get the user from the authentication directive instead. The frontend service will also need to be updated not to insert the id.